controller/templates/statefulset.yaml

```diff
@@ -102,8 +102,8 @@ spec:
           securityContext:
             runAsUser: 1000
             runAsGroup: 100
-            fsGroup: 100
```

Comment:
`fsGroup` is not allowed in a container security context. Only at the pod level. See [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#securitycontext-v1-core).